### PR TITLE
Remove tags for test suites that are not associated with this implementations repo and update `README.md`

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses #85 and #89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,12 +91,15 @@ following form:
...
+both.
+Implementations may also contain no security (do not add a OAUTH2 or ZCAP
+section in that case).
 
-#### Testing locally
+### Testing locally
 
-If you need to add implementations for endpoints running locally you may add a config file in the root dir of your test project:
+If you need to add implementations for endpoints running locally you may add a
+config file in the root dir of your test project:
```
Comment (BigBlueHat) [inline README.md:102]: Is there anyway to test _only_ a locally running implementation? If so (or not), we should likely handle it as a separate issue/PR...but I keep wondering, so thought I'd ask.

Ideally, something short of removing all the other implementations from the list temporarily...

Comment (JSAssassin) [reply README.md:102]: Yes, there is. In the test suites you can add a config file `.vcApiTestImplementationsConfig.cjs` and then you can add your implementations running locally like this -
https://github.com/w3c-ccg/vc-api-test-suite-implementations/blob/main/README.md#testing-locally and update the tags in the tests to match the tag you specify in your config. And then run the test like you would run normally.

Comment (JSAssassin) [reply README.md]: Included those additional guidelines under the `Testing locally` section to make it clearer [168dbd1](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/91/commits/168dbd137004b5962a04517d88c695e3d5c13278)

Comment (BigBlueHat) [reply README.md]: @JSAssassin this is great! Can you expand on how this is done, though?
> update the tags in the tests to match the tag you specify in your config. And then run the test like you would run normally

Comment (JSAssassin) [reply README.md]: @BigBlueHat Updated the section and added example here - [03fc824](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/91/commits/03fc824d1000666761ec52fcff4c997db6469e5a).

File: README.md [hunk 2]
```diff
@@ -122,28 +125,117 @@ module.exports = [{
...
 
-* `Ed25519Signature2020` - This tag will run the [Ed25519 tests](https://github.com/w3c/vc-di-ed25519signature2020-test-suite) on your issuer and/or verifier.
+### Opting into a Test Suite
 
-* `StatusList2021` - This tag will run the [Status List 2021 tests](https://github.com/w3c-ccg/status-list-2021-test-suite) on your issuer and/or verifier.
+Please Note:
 
-* `did-key` - This tag will run the [DID Key Test Suite](https://github.com/w3c-ccg/did-key-test-suite) on your DID resolver endpoint.
+1. Tags serve as identifiers to determine which test suites to run on your
+issuers and verifiers. Only the first issuer/verifier found with a specific
```
Comment (aljones15) [inline README.md:142]: this is actually test specific in actuality the test suite will see all endpoints marked with a tag and there could be a test suite that wants to run on multiple endpoints with the same tag. The later test does not exist yet, but could happen such as picking on vc-api compatible endpoint at random for an interop test.

**Review by aljones15 — CHANGES_REQUESTED** [review]
This is good, but needs some clarification.

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md:142]: @aljones15 I am having trouble coming up with an alternative, could you please provide what you would prefer to see here.

File: README.md [hunk 3]
```diff
@@ -122,28 +125,117 @@ module.exports = [{
...
+  }, {
+    "id": "https://product.example.com/verifiers/z4Rq7N1lT6zVwFgXk8JYdCcKpU",
+    "endpoint": "https://product.example.com/verifiers/z4Rq7N1lT6zVwFgXk8JYdCcKpU/credentials/verify",
+    "tags": ["vc-api"]
+  }]
+}
+```
+
+
+2. If an issuer or verifier supports multiple VC signature types, you can assign
```
[the remark this answers is not in the document]
Comment (JSAssassin) [reply README.md]: Updated the note here [f8bcfa1](https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/91/commits/f8bcfa18729698c31c1ab669b7002ef807cee01e), hopefully that addresses the issue.

File: README.md (hunk 2 above)
Comment (aljones15) [reply README.md:142]: 1. Tags serve as identifiers to determine which test suites to run on your
```suggestion
issuers and verifiers. Only the first issuer/verifier found with a specific
```
issuers and verifiers. So far the first issuer/verifier found with a specific

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (aljones15) [reply README.md]: perfect resolving this

**Review by aljones15 — APPROVED** [review]
thanks for the changes LGTM

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md]: @aljones15 Updated the wording to not include `Only` [03e6fe0](https://github.com/w3c-ccg/vc-test-suite-implementations/pull/91/commits/03e6fe00d618bf23edd8e0d681d7db76e6be37bb).

**Review by BigBlueHat — APPROVED** [review]
Excellent work, @JSAssassin!

**Pull request merged**